frontend/packages/ux-editor/src/components/config/editModal/EditCodeList.tsx

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
-import { TextField } from '@digdir/design-system-react';
+import { Select } from '@digdir/design-system-react';
 import { IGenericEditComponent } from '../componentConfig';
 import { useText } from '../../../hooks';
-import type { FormCheckboxesComponent, FormRadioButtonsComponent } from '../../../types/FormComponent';
+import { useOptionListIdsQuery } from '../../../hooks/queries/useOptionListIdsQuery';
+import { useParams } from 'react-router-dom';
+import { AltinnSpinner } from 'app-shared/components';
+import { ErrorMessage } from '@digdir/design-system-react';
+
+export enum SelectedOptionsType {
+  Codelist = 'codelist',
+  Manual = 'manual',
+  Unknown = '',
+}
 
 export function EditCodeList({ component, handleComponentChange }: IGenericEditComponent) {
   const t = useText();
+  const { org, app } = useParams();
 
+  const {
+    data: optionListIds,
+    isLoading: isLoadingCodeList,
+    error: isError,
+  } = useOptionListIdsQuery(org, app);
```

Comment:
`isError` er allerede returnert i hooken. Ikke forveksle den med `error`, som er selve objektet som blir kastet når det oppstår en feil. Her er en oversikt over hele objektet som blir returnert: https://tanstack.com/query/v4/docs/react/reference/useQuery

Det er bra å bruke spesifikke navn, men `isLoadingCodeList` er noe misvisende. Vi henter ikke en kodeliste, men en liste over ID-er til flere kodelister. Her er imidlertid spørringen såpass tett knyttet til komponenten, så det greier seg sikkert å bruke `isLoading` direkte, som `isError` og `error`.
```suggestion
  const {
    data: optionListIds,
    isLoading,
    isError,
    error
  } = useOptionListIdsQuery(org, app);
```